webextension/js/popup.js

```diff
@@ -64,38 +76,33 @@ function showContainerTabsPanel(identity) {
 }
 
 if (localStorage.getItem("onboarded2")) {
-  for (let element of document.querySelectorAll(".onboarding")) {
-    element.classList.add("hide");
-  }
-  document.querySelector("#container-panel").classList.remove("hide");
+  showPanel("#container-panel");
 } else if (localStorage.getItem("onboarded1")) {
-  document.querySelector(".onboarding-panel-1").classList.add("hide");
-  document.querySelector("#container-panel").classList.add("hide");
+  showPanel(".onboarding-panel-2");
 } else {
-  document.querySelector(".onboarding-panel-2").classList.add("hide");
-  document.querySelector("#container-panel").classList.add("hide");
+  showPanel(".onboarding-panel-1");
 }
 
 document.querySelector("#onboarding-next-button").addEventListener("click", () => {
   localStorage.setItem("onboarded1", true);
-  document.querySelector(".onboarding-panel-2").classList.remove("hide");
-  document.querySelector(".onboarding-panel-1").classList.add("hide");
-  document.querySelector("#container-panel").classList.add("hide");
+  showPanel(".onboarding-panel-2");
```

Comment:
This could be further simplified by calling a `checkCurrentStatus` method which could have the code above allowing the above code to call the same code also:
```
 if (localStorage.getItem("onboarded2")) {
```